runtime/lua/vim/diagnostic.lua

```diff
@@ -853,6 +850,13 @@ function M.goto_next(opts)
   return diagnostic_move_pos(opts, M.get_next_pos(opts))
 end
 
+local get_fallback_sign = function (severity)
```

Comment:
Minor nit but I think we prefer the `local function` syntax, at least in this file. The `---@private` annotation is needed to prevent documentation generation for this function.

```suggestion
---@private
local function get_fallback_sign(severity)
```